styles.css

```
@@ -0,0 +1,944 @@
+    outline-offset: 2px;
+}
+
+/* Print Styles */
+@media print {
+    .header,
+    .footer {
+        display: none;
+    }
+
+    .product-card {
+        page-break-inside: avoid;
```

Stylelint (17.6.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)